conf/messages.en

```diff
@@ -880,6 +880,10 @@ trustee.organisation.addressYesNo.checkYourAnswersLabel = Do you know {0}’s ad
 trustee.organisation.addressUkYesNo.checkYourAnswersLabel = Is {0}’s address in the UK?
 trustee.organisation.ukAddress.checkYourAnswersLabel = What is {0}’s address?
 trustee.organisation.internationalAddress.checkYourAnswersLabel = What is {0}’s address?
+trustee.organisation.5mld.countryOfResidence.checkYourAnswersLabel = What is {0}’s country of residency?
+trustee.organisation.5mld.countryOfResidenceInTheUkYesNo.checkYourAnswersLabel =  Does {0} have UK residency?
```

Comment:
Very minor but there's a double space after the equals sign. I suspect it doesn't but I'm never sure if that prepends a space to the message or not when it's rendered on the screen